modules/dependabot/templates/dependabot.yaml.tftpl

```diff
@@ -2,8 +2,10 @@ version: 2
 
 updates:
 %{ for key, value in ecosystems }
-  - package-ecosystem: ${value}
+  - package-ecosystem: "${value}"
     directory: "/"
     schedule:
       interval: "daily"
+    commit-message:
+      prefix: feat
```

Comment:
Does this mean it will always have feat prefix? also shouldn't it start with "feat:"?